src/styles/App.css

```diff
@@ -0,0 +1,21 @@
+@import "./book.css";
+@import "./progress.css";
+@import "./addbook.css";
+
+* {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  text-decoration: none;
+}
+
+body {
+  font-family: 'Roboto', sans-serif;
+  background-color: #fafafa;
+}
+
+#root {
+  align-items: center;
+
+  /* margin: auto; */
```

Comment:
[OPTIONAL] Please remove unwanted commented codes from your codebase because it's your codebase make messy if you want to check your precious code then check GitHub commit history :slightly_smiling_face: